coreclient/src/clients/mod.rs

```diff
@@ -568,9 +568,14 @@ impl CoreUser {
             .ds_send_message(params, group.leaf_signer(), group.group_state_ear_key())
             .await?;
 
-        // Phase 3: Merge the commit into the group
-        let connection = self.connection.lock().await;
+        // Phase 3: Merge the commit into the group & update conversation
+        let mut connection = self.connection.lock().await;
         group.store_update(&connection)?;
+        conversation.update_last_used(&connection)?;
+        Conversation::mark_as_read(
+            &mut connection.transaction()?,
```

Comment:
I don't think it works that way. You have to create the transaction before calling the function, then call the function and then commit the transaction.